python/tskit/trees.py

```diff
@@ -10476,6 +10476,45 @@ def ld_matrix(
             mode=mode,
         )
 
+    def individual_nodes(self):
```

Comment:
For backwards compatibility, we should provide a ``ploidy`` argument here I think which kicks in for the no-individuals case (and it's an error is this is not None when num_individuals > 0). Here, we do what the VCF code is currently doing.

Reply:
Really? I'm probably missing something, but that is long-ago-enough that I think it would only confuse things?

Reply:
Yes, I've changed my mind on this - see point below about making ``ts.individuals_nodes`` an attribute array.